nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/index.js

```diff
@@ -58,9 +73,9 @@ exports.wrapSequelize = (sequelize, include={}) => {
             comments.route = req.route.path;
         }
 
-        if (tracer.active) {
-            toW3CTraceContext(tracer.currentRootSpan, comments);
-        }
+        // Add trace context to comments, depending on the current provider.
+        const traceContext = traceProvider.getW3CTraceContext();
+        Object.assign(comments, traceContext);
```

Comment:
I think we should abstract this setup into the code in provider.js or even util.js itself thus, in  the endhere, all that we'll need to do inside this file is perhaps
```javascript
provider.attachComments(options.TraceProvider, comments);
```